build.gradle.kts

```diff
@@ -54,6 +62,56 @@ subprojects {
     version = LIBRARY_VERSION
 }
 
-tasks.register<Delete>("clean") {
+tasks.getByName<Delete>("clean") {
     delete(rootProject.layout.buildDirectory)
 }
+
+// Merges individual module docs into a single HTML output
+dependencies {
+    dokka(project(":core:"))
+    dokka(project(":connectors:supabase"))
+}
+
+dokka {
+    moduleName.set("PowerSync Kotlin")
+}
+
+// Serve the generated Dokka documentation using a simple HTTP server
+// File changes are not watched here
+tasks.register("serveDokka") {
+    dependsOn("dokkaGenerate")
```

Comment:
Let's add a `group = "dokka"` line here so that it shows up next to the other tasks in IDEs:

![Task overview](https://github.com/user-attachments/assets/31e82faa-d4b1-427d-90b6-4e38f08691b7)

Reply:
Added a group.